examples/diff.c

```diff
@@ -47,6 +47,15 @@ enum {
 	CACHE_NONE = 2
 };
 
+#define DEFAULT_THRESHOLD 50
+#define DEFAULT_BREAK_REWRITE_THRESHOLD 60
+#define USE_DEFAULT(X) ((X) == 0 || (X) > 100)
+
+struct pair {
+	int first;
+	int second;
+};
```

Comment:
There is no library. Each application has their own needs for such a function. You can look at git or other projects for some samples, but such a function generally simply prints whatever went wrong to `stderr` and exits the program with `1` or sometimes a more specific error code, depending on what the program does.